samples/peripheral/radio_test/sample.yaml

```diff
@@ -10,11 +10,12 @@ tests:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpunet
       - nrf7002dk/nrf5340/cpunet
+      - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
       - nrf54h20dk/nrf54h20/cpurad
     platform_allow: >
       nrf52dk/nrf52832 nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpunet nrf7002dk/nrf5340/cpunet
-      nrf54l15pdk/nrf54l15/cpuapp nrf54h20dk/nrf54h20/cpurad
+      nnrf54l15dk/nrf54l15/cpuapp rf54l15pdk/nrf54l15/cpuapp nrf54h20dk/nrf54h20/cpurad
```

Comment:
One of "n" is moved from pdk to dk target

Reply:
Fixed.